core/nginx/config.py

```diff
@@ -5,8 +5,7 @@
 import sys
 from socrate import system, conf
 
-system.set_env()
-args = os.environ.copy()
+args = system.set_env()
```

Comment:
I think you should also replace all "os.getenv()" with "args.get()" now.
Sorry, for not pointing this out.
This doesn't make a diference now, but will surely cause side-effects later.